code401challenges/src/test/java/code401challenges/sortMerge/MergeSortTest.java

```diff
@@ -0,0 +1,42 @@
+package code401challenges.sortMerge;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class MergeSortTest {
+
+    @Test
+    public void testMergeSort_basic() {
+        int[] expectedArr = new int[] {4, 8, 15, 16, 23, 42};
+        int[] actualArr = new int[] {8, 4, 23, 42, 16, 15};
+        MergeSort.mergeSort(actualArr);
+        assertEquals(Arrays.toString(expectedArr), Arrays.toString(MergeSort.mergeSort(actualArr)));
```

Comment:
You want to test these arrays directly, without using toString. To do that you just change your assert statement to: `assertArrayEquals(expectedArr, actualArr);`